service/handlers.go

```diff
@@ -0,0 +1,55 @@
+package service
+
+import (
+	"encoding/json"
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	"github.com/rancher/webhook-service/drivers"
+	"github.com/rancher/webhook-service/webhooks"
+	"io/ioutil"
+	"net/http"
+)
+
+type WebhookRequest map[string]interface{}
+
+func ConstructPayload(w http.ResponseWriter, r *http.Request) {
+	var webhookRequestData WebhookRequest
+	log.Infof("Construct Payload")
+	bytes, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		log.Errorf("Construct failed with error: %v", err)
+		return
+	}
+	json.Unmarshal(bytes, &webhookRequestData)
+	webhookRequestData["projectId"] = r.Header.Get("X-API-ACCOUNT-ID")
```

Comment:
I'm concerned with security around ppl injecting projectIDs into the payload. If we treat it special and keep it out of the payload, it eliminates that concern.